schedule/yast/ext4/ext4@yast.yaml

```diff
@@ -27,3 +27,12 @@ schedule:
   - installation/reboot_after_installation
   - installation/grub_test
   - installation/first_boot
+  - console/validate_ext4_fs
+test_data:
+  partitions:
+    - size: '7.2G'
```

Comment:
I'm not sure if we can validate size of the partition with this precision in scalable way, but we will see when will try this module on all architectures.